change-beta/@azure-communication-react-2839d297-3f9b-45d1-8db6-58f7175c5f19.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "prerelease",
```

Comment:
This should be `patch` or `none` as it applies to stable releases, I'd recommend `none` as we don't include sample changes in the changelog 
@prprabhu-ms we should add this detail to https://github.com/Azure/communication-ui-library/blob/main/docs/references/release-changelogs.md